.github/workflows/build_packages.yaml

```diff
@@ -16,21 +16,20 @@ jobs:
         strategy:
             matrix:
               os:
+                - ubuntu20.04
                 - ubuntu18.04
                 - ubuntu16.04
-                - ubuntu14.04
                 - debian10
                 - debian9
-                - debian8
-                - opensuse
+                - centos8
                 - centos7
                 - centos6
```

Comment:
Del centos6?

Reply:
this one works fine for now, so I'll leave it alone.